integration-tests/tests/src/performance-tests/property-reads.ts

```diff
@@ -84,27 +79,27 @@ function describeTypeRead({ type, value, schema = [] }: TestParameters) {
 }
 
 const cases: Array<TestParameters | [Realm.PropertySchemaShorthand | Realm.PropertySchema, Value]> = [
+  ["bool?", true],
   ["bool", true],
+  ["int?", 123],
   ["int", 123],
   ["float", 123.456],
+  ["double?", 123.456],
   ["double", 123.456],
-  ["string", "Hello!"],
-  ["decimal128", new Realm.BSON.Decimal128("123")],
-  ["objectId", new Realm.BSON.ObjectId("0000002a9a7969d24bea4cf4")],
-  ["uuid", new Realm.BSON.UUID()],
-  ["date", new Date("2000-01-01")],
-  ["data", new Uint8Array([0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09])],
+  ["string?", "Hello!"],
```

Comment:
I'm keeping everything else using optional to simplify comparisons with v11 perf for now.